rasa/shared/core/domain.py

```diff
@@ -1524,6 +1542,11 @@ def intents(self) -> List[Text]:
         """Returns sorted list of intents."""
         return sorted(self.intent_properties.keys())
 
+    @rasa.shared.utils.common.lazy_property
+    def entities(self) -> List[Text]:
```

Comment:
We also need a setter method for entities, as highlighted by this test failure `test_check_for_unseen_feature`.

Reply:
If it's the only place it's set from outside, I'm wondering if it's not better to change the test - the parallel intents property is not settable from outside and in general seems better not to make a derived property settable